pkg/workflows/wasm/host/module.go

```diff
@@ -88,6 +89,7 @@ type ModuleConfig struct {
 	InitialFuel    uint64
 	Logger         logger.Logger
 	IsUncompressed bool
+	Fetch          func(wasm.TargetRequestPayload) (wasm.TargetResponsePayload, error)
```

Comment:
Nit can we name this FetchRequest and FetchResponse: Target is a type of capability and is not relevant in this context